src/main/java/graphql/execution/ExecutorServiceExecutionStrategy.java

```diff
@@ -58,8 +72,9 @@ public ExecutorServiceExecutionStrategy(ExecutorService executorService, DataFet
             ExecutionStrategyParameters newParameters = parameters
                     .transform(builder -> builder.field(currentField).path(fieldPath));
 
-            Callable<ExecutionResult> resolveField = () -> resolveField(executionContext, newParameters).join();
-            futures.put(fieldName, executorService.submit(resolveField));
+            Callable<ExecutionResult> fieldResolver = () -> resolveField(executionContext, newParameters).join();
+            Callable<ExecutionResult> wrappedFieldResolver = callableWrapper.wrapCallable(fieldResolver);
+            futures.put(fieldName, executorService.submit(wrappedFieldResolver));
         }
```

Comment:
For an example -- it may be reasonable for the `ExecutorServiceExecutionStrategy` implementation to switch to using `ExectorService#invokeAll(Collection<? extends Callable<T>>)` instead of "manually" using `#submit(Callable)` and `Future#get()` for each field.